agent/consul/server_test.go

```diff
@@ -115,6 +117,10 @@ func testServerConfig(t *testing.T) (string, *Config) {
 	return dir, config
 }
 
+func MakeMeATestServer(t *testing.T) (string, *Server) {
```

Comment:
This appears to not be used.

Reply:
That's very odd. I would swear I did not write that func. Maybe it was part of someone's test code that got merged in while rebasing. I'll take it out.

Reply:
Fixed by 626437f553906b955871b0353930cb56aaeeeed5.